### ISSUE TITLE
Upgrading from 0.14.0-b1 to 0.14.0-b3 fails OTA

### ISSUE THREAD

**Issue by Sdub76** [scottrbailey/WLED-Utils#13]
Tried updating my SP511e with this updated firmware and get the error: Update failed! Please check your file and retry!

https://github.com/srg74/WLED-wemos-shield/blob/master/resources/Firmware/%40Aircoookie/Latest/WLED_0.14.0-b3_sp511e_2m.bin

Anyone else have luck with this update? Am I pulling the wrong binary?

**Comment by scottrbailey** [comment]
No.  At 0.13.x the size of the binary was right on the cusp of being able to do OTA updates on 2MB devices.  There were some work arounds to do OTA update on the ESP01.  But I'm afraid the days of OTA updates on 2MB devices are largely over.

**Comment by Sdub76** [comment]
Does that mean you will need to manually flash these updates each time, or am I just stuck on 0.14.0-b1 forever?

**Comment by scottrbailey** [comment]
No, you can update manually.  Just not OTA.

**Comment by RobertoD91** [comment]
Same "issue," I can't update from version 13 to 14 using OTA; I have to flash it using a serial connection.
Perhaps the page https://github.com/scottrbailey/WLED-Utils/blob/gh-pages/sp511e_wled.md could be updated to include this information?

PS: Tasmota still uses OTA updates; why can't WLED?
